### ISSUE TITLE
FASTRCNN example Pascal dataset 2007 (mAP) at around 0.4565

### ISSUE THREAD

**Issue by pmfcdb** [Microsoft/CNTK#998]
Hello,
I got the example FASTRCCN Pascal dataset 2007 running, but the (mAP) is at around 0.4565, with my trained model and with yours. Is this the actual mAP? What can I do to improve?

Using CNTK-2-0-beta1-0-windows-64bit-GPU, laptop i7-6700 2.60GHz 16GB Ram and nvidia gtx 970M windows 10 home

I saw in this site : https://github.com/ShaoqingRen/faster_rcnn that VOC 2007 test was around 69.9%

I posted a video in youtube recording every steps, you can see everything, from python version, to pip list, script execution, also script B2_EvaluateInputROIs.py doesn't work because of path of subdirs(positive and testImages), and missing *.tsv files.

https://youtu.be/EaB8RjH4oGY

04:10 Visualization Input ROIs
06:58 (mAP) at around  0.4565
07:13 Visualization Output ROIs
Thks,

**Comment by cha-zhang** [comment]
One major difference is that we are using AlexNet, while the original implementation uses VGG16. Also, we are still implementing bounding box regression, which should add a few points to the mAP score. 

We are actively working on this. Please stay tuned.

**Comment by pmfcdb** [comment]
Any news regarding this subject?

**Comment by cha-zhang** [comment]
Sorry, not yet. We are building a Caffe->CNTK converter to get the same VGG models used by the paper. It has taken a while and we are still working on it.

**Comment by pmfcdb** [comment]
Hello, I implement a small dataset ( similar to the grocery dataset) and I achieve 0,6358 mAP with beta11.

Will you use in the future VGG16 instead of AlexNet?
In the CsEval code : 
....
 // We can call the evaluate method and get back the results (predictions per ROI and per class (no softmax applied yet!)...
                    var outDims = model.GetNodeDimensions(NodeGroup.Output);
                    outputs = model.Evaluate(inputs, outDims.First().Key);
....
Will there be suport for softmax in CsEval?

Thks,

**Comment by cha-zhang** [comment]
Yes, we will be release converted VGG16 very soon. 

Softmax is only done if you want to compute probability. In that case, you can do it yourself. If you just take argmax of the output, it is not needed.

**Comment by pmfcdb** [comment]
Hi, is the Caffe->CNTK model converter being released also? I would like to try these models from caffe:

http://www.robots.ox.ac.uk/~vgg/software/vgg_face/

https://github.com/AlfredXiangWu/face_verification_experiment

Paulo Brito

**Comment by pkranen** [comment]
You can see the current state of the converter in the branch v-yuxgu/model2cntk. I'll close this issue since the topic diverged from the original subject. Feel free to open a new one related to the converted is required.
